README.rst

```diff
@@ -84,17 +90,37 @@ Launch Sphinx in an ephemeral pex environment using the Sphinx entry point ``sph
 
     $ pex sphinx -e sphinx:main -- --help
 
-Build a standalone pex binary into ``pex.pex`` using the ``pex`` console_scripts entry point:
+Using ``setup.py`` Entry Points
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have a ``pex_entry`` entry point in the `console_scripts <https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point>`_ section of your ``setup.py``:
+
+.. code-block:: python
+
+    setup(
+        ...
+        entry_points = {
+            'console_scripts': [
+                'pex_entry=my_package.my_module:main',
+            ],
+        }
+        ...
+    )
+
+you can build a standalone pex binary, including the ``pex`` package itself, into ``pex-executable.pex`` using the ``pex_entry`` entry point:
 
 .. code-block:: bash
 
-    $ pex pex -c pex -o pex.pex
+    $ pex pex -c pex_entry -o pex-executable.pex
```

Comment:
The `pex_entry` console script is not a valid entrypoint for the pex distribution in any release. How about:
```
$ pex "pex>=2.1.35" -c pex-tools ...
```
The pex-tools console script is not confusingly overloading the `pex` word and it is available starting only in Pex 2.1.35 which forces a `>=...` which further disambiguates tokens.

Reply:
And that's defined here btw:
https://github.com/pantsbuild/pex/blob/c92de90f5265c074ab66a225ad08d96ad045eccd/pyproject.toml#L40-L42

Reply:
Between setup.py, Poetry, PEP-621 and this, you should now be doubled down on your python packaging ecosystem frustration!

Reply:
Another thing to consider is using `--console-script` instead of `-c` and `--output-file` instead of `-o` in this 1 example, say, and then flipping to the the short forms. Those explicit flags undoubtedly also help understand what is what.

Reply:
Good call on all of those points. I'm a seasoned developer and I'm still confused and frustrated with Python packaging! 😆

I'll make those changes.